types/format-thousands/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for format-thousands 2.0
+// Project: https://github.com/VovanR/format-thousands
+// Definitions by: Jerry Wang <https://github.com/c1495616js>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = format_thousands;
+
+declare function format_thousands(number?: number | string, option?: string | { separator?: string; formatFourDigits?: boolean }): string;
```

Comment:
```suggestion
export = formatThousands;

declare function formatThousands(number?: number | string, option?: string | { separator?: string; formatFourDigits?: boolean }): string;
```
it's named `formatThousands` (just to stick to the JS convention for naming things):
https://github.com/VovanR/format-thousands#usage

Reply:
Thanks @peterblazejewicz for the review.
Addressed it.